draft-cms-masque-ip-proxy-reqs.md

```diff
@@ -114,6 +114,16 @@ without a VPN client installed on the user's device. This style of connectivity
 can also be used to connect devices that cannot run VPN clients through to the
 network.
 
+## Consumer VPN
```

Comment:
While this is all true, I consider this a special-case of point-to-network connectivity. I would suggest moving it above, or merging it.

Reply:
I moved it to the top - I think keeping them separate will make it clearer, since it wasn't obvious to EKR originally.